src/screens/FearConditioningTrialScreen.tsx

```diff
@@ -78,6 +78,7 @@ export const FearConditioningTrialScreen: React.FunctionComponent<FearConditioni
     const ratingValueRef = useRef<any>()
     const headphoneSensorListener = useRef<EmitterSubscription>()
     const headphoneRef = useRef<AlertRef>()
+    const headphoneIsConnectedRef = useRef<boolean>()
```

Comment:
Note: Theres a few different effects and state changes in this component, a useRef just seemed easier to keep track of this value when mangaing it with alerts.